DFC.ServiceTaxonomy.Theme/wwwroot/Styles/trumbowyg_scoped_govuk_frontend.scss

```diff
@@ -110,3 +124,15 @@
 
     @include meta.load-css("../../node_modules/nationalcareers_toolkit/assets/dist/css/all");
 }
+
+// TO DO: Remove the following two @includes when @font-face:src/url updates in nationalcareers_toolkit
+// currently the nationalcareers_toolkit mapping src url at  /nationalcareers_toolkit/fonts/
+// but expecting the src url to map at : /nationalcareers_toolkit/fonts/assets/dist/fonts
+@include fontface($font-face-light-url-woff,$font-face-light-url-woff2,normal);
+@include fontface($font-face-bold-url-woff, $font-face-bold-url-woff2,bold);
+
+.trumbowyg-modal-box {
+    label {
+        width: calc(100% - 20px);
```

Comment:
Does this apply to all trumbowyg modal boxes?

If so, is this width appropriate for all of its modal boxes?

If not, can we make it select just the youtube link modal?

Reply:
Yes! it's applies all modal boxes.
Yes! new changes works very for all scenarios